inventory/hosts.example

```diff
@@ -637,6 +637,36 @@ debug_level=2
 #openshift_persistentlocalstorage_path=/mnt/local-storage
 #openshift_persistentlocalstorage_provisionner_image=quay.io/external_storage/local-volume-provisioner:v1.0.1
 
+# Cluster monitoring
+#
+# Cluster monitoring is enabled by default, disable it by setting
+# openshift_cluster_monitoring_operator_install=false
+#
+# Cluster monitoring configuration variables allow setting the amount of
+# storage requested through PersistentVolumeClaims, which node-selector to use
+# to run components of cluster-monitoring on and the configuration to use for
+# the Prometheus Alertmanager.
+#
+# openshift_cluster_monitoring_operator_prometheus_storage_capacity: "50Gi"
+# openshift_cluster_monitoring_operator_alertmanager_storage_capacity: "2Gi"
+# openshift_cluster_monitoring_operator_node_selector: "{{ openshift_hosted_infra_selector | default('node-role.kubernetes.io/infra=true') | map_from_pairs }}"
+# openshift_cluster_monitoring_operator_alertmanager_config: |+
```

Comment:
This probably should be dropped - its too complex to rewrite in INI format and probably should go to openshift-docs instead